detect_secrets/core/usage.py

```diff
@@ -355,6 +346,15 @@ def add_arguments(self):
 
         self._add_report_module()
 
+        self.parser.add_argument(
+            'filename',
+            nargs='+',
+            help=(
+                'Audit a given baseline file to distinguish the difference '
+                'between false and true positives.'
+            ),
+        )
+
```

Comment:
Allows for `detect-secrets audit --report .secrets.baseline --fail-on-xx` by moving the filename arg to the end. Before, it would have had to be `detect-secrets audit .secrets --report --fail-on-xx`, which is just confusing.

Yelp's DS follows the same pattern, filename is at the end: https://github.com/Yelp/detect-secrets/blob/master/docs/audit.md#report-generation.